source/dub/packagemanager.d

```diff
@@ -899,7 +899,7 @@ symlink_exit:
 		this.m_repositories[type].writeLocalPackageList();
 
 		foreach(ver, name; removed)
-			logInfo("Deregistered package: %s (version: %s)", name, ver);
+			logInfo("De-registered package: %s (version: %s)", name, ver);
```

Comment:
Maybe we should use "unregistered" (or "un-registered") here ?

Reply:
went with Deregistered after all: https://english.stackexchange.com/questions/25931/unregister-vs-deregister